File: pyg_seckill_interface/src/main/java/com/pyg/seckill/service/SeckillService.java
package com.pyg.seckill.service;

import com.pyg.pojo.TbSeckillGoods;

import java.util.List;

public interface SeckillService {
    List<TbSeckillGoods> findAllSeckillGoodsFromRedis();

    TbSeckillGoods findOneSeckillGoodsFromRedis(Long id);

    void saveOrder(Long id,String userId);
}

File: pyg_seckill_web/src/main/webapp/js/service/seckillService.js
app.service("seckillService",function ($http) {

    this.findSeckillGoods=function () {
        return $http.get("./seckill/findSeckillGoods");
    }
    this.findOne=function (id) {
        return $http.get("./seckill/findOne?id="+id);
    }
    this.saveOrder=function (id) {
        return $http.get("./seckill/saveOrder?id="+id);
    }

})